lib/createsend.js

```diff
@@ -46,17 +47,22 @@ function CreateSend (auth, options) {
             });
         },
 
-        request: function (method, path, qs, params, callback) {
-            var uri = ((this.secure) ? 'https' : 'http') + '://' +
-                this.baseUri + '/api/' + this.version + '/' + path;
+        request: function (method, path, qs, params, callback, txBool) {
+            var uri;
+            if(txBool===false) {
+                uri = ((this.secure) ? 'https' : 'http') + '://' + this.baseUri + '/api/' + this.version + '/' + path;
+            }
+            else {
+                uri =  ((this.secure) ? 'https' : 'http') + '://' + this.baseUri + '/api/' + 'v3.1' + '/' + path;
+            }
```

Comment:
There's a bit of redundancy here that feels a bit uncomfortable to me. I would prefer to see something more like

    var uri = (this.secure ? 'https' : 'http') + '://' + this.baseUri + '/api/' + 
        (txBool === false ? this.version : 'v3.1') + '/' + path;

I think it makes the bit that depends on txBool a bit clearer here.